native/cocos/renderer/gfx-base/GFXBuffer.h

```diff
@@ -68,6 +93,7 @@ class CC_DLL Buffer : public GFXObject, public RefCounted {
     uint32_t _size = 0U;
     uint32_t _offset = 0U;
     BufferFlags _flags = BufferFlagBit::NONE;
+    uint8_t *_data{nullptr};
```

Comment:
We can put _data after _isBufferView. This change might reduce size of one pointer in 64bit platform.